src/home/quote/state/quoteReducer.spec.js

```diff
@@ -1,5 +1,4 @@
 import expect from 'expect';
```

Comment:
This file also shouldnt be here

The only test he has so far is `empty.spec.js`